presto-main/src/main/java/io/prestosql/sql/rewrite/ShowStatsRewrite.java

```diff
@@ -140,9 +140,23 @@ protected Node visitShowStats(ShowStats node, Void context)
                 throw rewriteAccessDeniedException(e);
             }
 
+            Table table = getTable(node, specification);
+            QualifiedObjectName tableName = createQualifiedObjectName(session, node, table.getName());
+            TableHandle tableHandle = metadata.getTableHandle(session, tableName)
+                    .orElseThrow(() -> semanticException(TABLE_NOT_FOUND, node, "Table '%s' not found", table.getName()));
```

Comment:
Use the `getTableHandle(Node node, QualifiedName table)` method, instead of removing it
https://github.com/prestosql/presto/blob/1f9088dc5757d18abf1f622f64d8bad4a87e8e2b/presto-main/src/main/java/io/prestosql/sql/rewrite/ShowStatsRewrite.java#L263-L268

Reply:
I am going to use `tableName` later on, so there are not much benefits of this method.